sqlmesh/core/context.py

```diff
@@ -541,12 +541,12 @@ def snapshots(self) -> t.Dict[str, Snapshot]:
         snapshots = {}
 
         for model in models.values():
+            config = self.config_for_model(model)
```

Comment:
Why was this moved here?

Reply:
for an earlier version. Looks like it can be moved back now.